### ISSUE TITLE
JSON List of Demos

### ISSUE THREAD

**Issue by jsloyer** [IBM-Bluemix/list-of-demos#1]
The Bluemix Code Team needs a JSON feed of curated demos that we can suck into Bluemix Code and display them in Bluemix Code.

/cc @triceam, @jakepeyser

**Comment by uwefassnacht** [comment]
Yes, agreed. @antonmc and @Twanawebtech are working on just that. This repo here is meant as a temporary list of demos and it will be replaced by http://ibm-bluemix.github.io once
